api/package.json

```diff
@@ -96,6 +96,7 @@
     "@types/exceljs": "1.3.0",
     "@types/express": "~4.17.17",
     "@types/jest": "~29.5.3",
+    "@types/lodash": "4.17.5",
```

Comment:
This resolves this recent lodash types issue - something I removed must have been giving us this for free https://github.com/DefinitelyTyped/DefinitelyTyped/issues/69932